front/src/modules/ui/input/components/IconPicker.tsx

```diff
@@ -0,0 +1,80 @@
+import { useMemo, useState } from 'react';
+import styled from '@emotion/styled';
+
+import { LightIconButton } from '@/ui/button/components/LightIconButton';
+import { DropdownMenuSearchInput } from '@/ui/dropdown/components/DropdownMenuSearchInput';
+import { StyledDropdownMenu } from '@/ui/dropdown/components/StyledDropdownMenu';
+import { StyledDropdownMenuItemsContainer } from '@/ui/dropdown/components/StyledDropdownMenuItemsContainer';
+import { StyledDropdownMenuSeparator } from '@/ui/dropdown/components/StyledDropdownMenuSeparator';
+import { IconComponent } from '@/ui/icon/types/IconComponent';
+
+type IconPickerProps = {
+  icons: Record<string, IconComponent>;
+  onChange: (iconName: string) => void;
+  selectedIconName?: string;
```

Comment:
selectedIconKey
(let's use key and not name that could be confused with a "displayed" name)

Reply:
Done!